lib/api/core/models/security/user.js

```diff
@@ -1,31 +1,70 @@
+var
+  q = require('q'),
+  async = require('async'),
+  _ = require('lodash');
+
 /**
  * @constructor
  */
 function User () {
   this._id = null;
-  this.profileId = null;
+  this.profilesIds = [];
 }
 
 /**
  * @param {Kuzzle} kuzzle
  *
  * @return {Promise}
  */
-User.prototype.getProfile = function (kuzzle) {
-  return kuzzle.repositories.profile.loadProfile(this.profileId);
+User.prototype.getProfiles = function (kuzzle) {
+  return kuzzle.repositories.profile.loadProfiles(this.profilesIds);
 };
 
 /**
  * @return {Promise}
  */
 User.prototype.getRights = function (kuzzle) {
-  return this.getProfile(kuzzle)
-    .then(profile => profile.getRights(kuzzle));
+  return this.getProfiles(kuzzle)
+    .then(profiles => {
+      var promises = profiles.map(profile => {
+        return profile.getRights(kuzzle);
+      });
+      
+      return q.all(promises)
+        .then(results => {
+          var rights = {};
+
+          results.forEach(right => {
+            _.assign(rights, right);
```

Comment:
be careful, if a first profile returns a right item with value "allowed" for a controller/action/index/collection tuple, and a second profile returns a right item with value "conditional", here, you will get the value "conditional", while we want the value "allowed".

For each right item, we want to be sure to have following values:
* "allowed" if at least a profile right has "allowed" value
* else,  "conditional" if at least a profile right has "conditional" value
* else, "denied"

For that, this should be replace by something similar to `_.assignWith(rights, right, mergeRights)`, like in Profile prototype (see https://github.com/kuzzleio/kuzzle/blob/develop/lib/api/core/models/security/profile.js#L123 ), using the `mergeRights` method defined there: https://github.com/kuzzleio/kuzzle/blob/develop/lib/api/core/models/security/profile.js#L142 (that should be moved away from Profile prototype to a shared piece of code, I guess).